src/js/bellows.js

```diff
@@ -52,6 +52,12 @@
             this._bindEvents();
         },
 
+        destroy: function() {
+            this.$bellows
+                .removeData(this.name)
+                .off(this.options.event);
```

Comment:
Ya namespacing would be my preferred approach. 

Only thing is if someone added an overridden event, it would unbind all of those.